Makefile

```diff
@@ -358,25 +347,20 @@ deploy-service-to-minikube:
 
 # Delete all the services from Minikube
 delete-all-services-from-minikube:
-	$(foreach var,$(SERVICES),helm uninstall $(var);)
+	$(foreach var,$(SERVICES),helm uninstall $(var) --ignore-not-found;)
 
 # Deploy All the services to minikube
 deploy-all-services-to-minikube:
 	$(foreach var,$(SERVICES),SERVICE=$(var) make deploy-service-to-minikube;)
 
-# bucketeer deploy
-deploy-bucketeer:
-	make -C ./ generate-tls-certificate
-	make -C ./ generate-oauth
-	make -C ./ service-cert-secret
-	make -C ./ oauth-key-secret
-	GITHUB_TOKEN=$(GITHUB_TOKEN) make -C ./ generate-github-token
-	ISSUER=$(ISSUER) \
-	EMAIL=$(EMAIL) \
-	OAUTH_KEY_PATH=/workspaces/bucketeer/tools/dev/cert/oauth-private.pem \
-	SERVICE_TOKEN_PATH=/workspaces/bucketeer/tools/dev/cert/service-token \
-	make generate-service-token
+# Bucketeer deployment
+deploy-bucketeer: delete-all-services-from-minikube
+	make -C tools/dev service-cert-secret
+	make -C tools/dev service-token-secret
+	make -C tools/dev oauth-key-secret
+	GITHUB_TOKEN=$(GITHUB_TOKEN) make -C tools/dev generate-github-token
```

Comment:
We can also remove the github token after merge db migration PR.

This github token is used by old database migration: https://github.com/bucketeer-io/bucketeer/blob/main/pkg/backend/cmd/server/server.go#L302.

Reply:
I will remove it later.